rmw_fastrtps_cpp/src/rmw_take.cpp

```diff
@@ -24,41 +24,51 @@
 extern "C"
 {
 rmw_ret_t
-rmw_take(const rmw_subscription_t * subscription, void * ros_message, bool * taken)
+rmw_take(
+  const rmw_subscription_t * subscription,
+  void * ros_message,
+  bool * taken,
+  rmw_subscription_allocation_t * allocation)
 {
+  (void) allocation;
```

Comment:
This shouldn't be necessary since we are using it by passing it through.  Same below.